### PR TITLE
Add mask type

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: sql/rikai--0.1.sql [hunk 1]
```diff
@@ -6,6 +6,14 @@ CREATE TYPE image AS (uri TEXT);
 
 CREATE TYPE detection AS (label TEXT, label_id int, box box, score real);
 
+-- rikai.types.Mask
+CREATE TYPE mask AS (
+    type INT,
```
Comment (changhiskhan) [inline sql/rikai--0.1.sql:11]: what's the type parameter again? whether it's RLE or polygon representation?

Comment (eddyxu) [reply sql/rikai--0.1.sql:11]: https://github.com/eto-ai/rikai/blob/main/python/rikai/types/geometry.py#L550-L555 
it is used differentiate 2 RLE types and polygons

**Review by changhiskhan — APPROVED** [review]
Should `type` be an `ENUM`?

**Comment by eddyxu** [comment]
Ok, changed to `Enum`

**Pull request merged**